sync.cmd

```diff
@@ -0,0 +1,3 @@
+@if "%_echo%" neq "on" echo off
+call executor.cmd %* sync -p
```

Comment:
The extra arguments are still being passed to the sync command. It looks confusing but it is because of how the command line parser behaves when you pass the `-?` option.
`call executor.cmd -? sync -p` only prints help for the sync command, while `call executor.cmd sync -p -?` prints help for all the commands.